ros2bag/ros2bag/verb/record.py

```diff
@@ -64,6 +64,8 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         # Topic filter arguments
         parser.add_argument(
             'topics', nargs='*', default=None, help='List of topics to record.')
+        parser.add_argument(
+            '--topic_types', nargs='+', default=[], help='List of topic types to record.')
```

Comment:
Please rename `--topic_types` parameter to the `--topic-types` for consistency and I think the corresponding Python parameter also expecting that it should be `--topic-types`.
```suggestion
        parser.add_argument(
            '--topic-types', nargs='+', default=[], help='List of topic types to record.')
```